src/TLuaInterpreter.cpp

```diff
@@ -1251,52 +1251,47 @@ int TLuaInterpreter::getMapEvents(lua_State * L){
     return 0;
 }
 
-int TLuaInterpreter::centerview( lua_State * L )
+int TLuaInterpreter::centerview(lua_State* L)
 {
-    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
-    if( ! pHost ) {
-        lua_pushnil( L );
+    Host* pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if (!pHost) {
+        lua_pushnil(L);
         lua_pushstring(L, "centerview: NULL Host pointer - something is wrong!");
         return 2;
-    }
-    else if( ! pHost->mpMap || ! pHost->mpMap->mpRoomDB ) {
-        lua_pushnil( L );
-        lua_pushstring(L, "centerview: no map present or loaded!");
+    } else if (!pHost->mpMap || !pHost->mpMap->mpRoomDB || !pHost->mpMap->mpMapper) {
+        lua_pushnil(L);
+        lua_pushstring(L, "centerview: you haven't opened a map yet");
```

Comment:
I actually feel pretty passionately about making software an easier and a friendlier experience and I've applied it with success in my Mudlet AI systems ([1](https://github.com/svof/svof/blob/a1f117f2e58a790bc63495b33d8339f949cef421/raw-svo.customprompt.lua#L522), [2](https://github.com/svof/svof/blob/3a1ca4da34ec8a3f4f202b05cbf3e1fd0fb9a130/raw-svo.actionsystem.lua#L116)) and years of field testing didn't reveal any issues with it. This isn't anything new either - just googling "error messages ux best practices", [first google result](https://conversionxl.com/error-messages/) talks about making them friendly, approachable, and actionable - something I think my message does.